build/templates/_library_interpreter.py/_get_error_description.py.mako

```diff
@@ -0,0 +1,21 @@
+    def get_error_description(self, error_code):
+        '''get_error_description
+
+        Returns the error description.
+        '''
+        try:
+            _, error_string = self.get_error()
+            return error_string
+        except errors.Error:
+            pass
+
+        try:
+            '''
+            It is expected for _get_error to raise when the session is invalid
+            (IVI spec requires GetError to fail).
+            Use _error_message instead. It doesn't require a session.
```

Comment:
I think we should remove the leading underscore from these names, with the move to library_interpreter.

Reply:
Done